docs/catalog/cloud/encrypt-what-is-necessary

```diff
@@ -0,0 +1,20 @@
+### Description
+Data protection through encryption is a crucial aspect of our security measures. However, the encryption process can be resource-intensive at multiple levels. Firstly, the amount of CPU required for encryption varies depending on the chosen algorithm, and more complex algorithms tend to demand higher computational power. Additionally, encryption can lead to increased storage requirements as it inflates the size of the data being stored because it typically contains additional metadata and padding, which is especially noticeable for smaller files. Furthermore, encryption is a repetitive task that needs to be performed each time data is fetched or updated. This repetitive nature can contribute to increased energy consumption, especially in high-throughput systems. Lastly, the most common approach involves encrypting the entire database or storage, without segregating sensitive and non-sensitive data. While this provides a high level of overall security, it also means that all data within the system incurs the energy cost associated with encryption, regardless of its sensitivity.
+
+### Solution
+1. Segregate data into sensitive and non-sensitive data and only encrypt the sensitive data.
+2. Revise policy of what is considered sensitive and understand which environment can impact this sensitivity.
+3. Choose the appropriate algorithm and the number of iterations in case of hashing.
+
+### SCI Impact
+SCI = (E * I) + M per R [Software Carbon Intensity Spec](https://grnsft.org/sci)
+Only encrypting what needs to be encrypted will impact SCI as follows:
+E: Reducing the amount of data that needs encryption will reduce the amount of energy needed to encrypt/decrypt, and store the data.
+
+### Assumptions
+We assume that a significant amount of data is not sensitive and can be stored unencrypted.
+
+### Considerations
+1. Client side encryption.
```

Comment:
Tbh, client side encryption would not change the energy requirement needed, doesn't it? Depending on where specialised hardware is available, this could also worsen it.